### PR TITLE
Enhance example installation command

### PR DESCRIPTION
**Pull request by ZhukovAlexander**
Make sure to return to the original directory after git clone

### CODE REVIEW

**Comment by davidhalter** [comment]
I just checked and it looks like this is supported by at least dash/bash/zsh. However I'm still not sure this is a good idea. People actually want to be in the directory where they installed something.

I'm new to `cd -`. Did never see it before, even if we don't merge it, thanks for the hint :)

@blueyed You can decide, I'm somewhat indifferent.

**Comment by ZhukovAlexander** [comment]
> I'm new to cd -

That is exactly why I added it :slightly_smiling_face:  I ended up in another directory after installation and didn't remember the exact original path.

File: README.rst [hunk 1]
```diff
@@ -89,7 +89,7 @@ Example installation command using Pathogen:
 
 .. code-block:: sh
 
-    cd ~/.vim/bundle/ && git clone --recursive https://github.com/davidhalter/jedi-vim.git
+    cd ~/.vim/bundle/ && git clone --recursive https://github.com/davidhalter/jedi-vim.git && cd -
```
Comment (blueyed) [inline README.rst]: What about `git clone --recursive https://github.com/davidhalter/jedi-vim.git ~/.vim/bundle/jedi-vim` instead?

Comment (ZhukovAlexander) [reply README.rst]: Wow, it totally slipped my mind. This is a perfect one.

**Comment by blueyed** [comment]
Thank you!

**Pull request merged**